package.json

```diff
@@ -25,6 +25,7 @@
     "build": "npm run clean && tsc && npm run test",
```

Comment:
hi @cliedeman,  for a better optimization I would ask you to modify the scripts in the package.json like this:
```
"scripts": {
    "clean": "rimraf lib",
    "compile": "npm run clean && tsc",
    "build": "npm run compile && npm run test",
    "test": "cross-env NODE_ENV=test jest --coverage",
    "format": "prettier --write \"src/**/*.{j,t}s*\"",
    "eslint": "eslint ./src --ext .js,.jsx,.ts,.tsx",
    "prepublishOnly": "npm run build"
 },
```

And to change the step and the **build** command to **compile**.

These changes avoid _compile_ and _test_ in the **install dependencies** step and _tests_ in the **build** step.

what do you think about it? :)

Reply:
No strong preference. Done